src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -108,7 +108,7 @@ const PIAFormPage = () => {
     key: keyof IPiaForm,
     isEager?: boolean,
   ) => {
-    // DO NOT allow state changes in the view mode
+    // DO NOT allow state changes in the view mode unless it is a review page
     if (mode === 'view' && !pathname?.split('/').includes('review')) return;
```

Comment:
I know this is not part of this ticket, but i somehow do not like this type of logic. seems very ugly.

Reply:
100%